mysys/mf_iocache.c

```diff
@@ -72,7 +86,7 @@ PSI_file_key key_file_io_cache;
     info		IO_CACHE handler
 
   NOTES
-    This is called on automaticly on init or reinit of IO_CACHE
+    This is called on automatically on init or reinit of IO_CACHE
```

Comment:
Strictly speaking this is exactly the kind of patch we don't merge

Reply:
reverted